migrations/20230619014713-create_private_sharing_folder_table.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('private_sharing_folder', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      folder_id: {
```

Comment:
We are deprecating auto-incremental ids. Let's point `private_sharing_folder.folder_id` to `folders.uuid`

Reply:
Remove this field if we are going to user folder_uuid, as in the future we will remove this field from folders